### DISCUSSION TITLE
Any plans to support other Charting libs ?

### DISCUSSION THREAD

**Discussion by Celibioux** [primefaces/community#1683] [PrimeVue] [answered]
I Know this is huge to ask but Charts.js uses only Html 5 Canvas for rendering and it's seems slow for some use cases there is Unovis lib and it uses WebGL and also Echarts js uses Svg Rendering
Thanks for your time reading this

**Comment by tugcekucukoglu** [comment, marked as answer]
In future, we are not going to plan 3rd party library. We are going to create our own Charts.

Thank you,
